info.portfolio_performance.PortfolioPerformance.json

```diff
@@ -0,0 +1,67 @@
+ {
+  "app-id": "info.portfolio_performance.PortfolioPerformance",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.38",
+  "sdk": "org.gnome.Sdk",
+  "sdk-extensions": [
+    "org.freedesktop.Sdk.Extension.openjdk11"
+  ],
+  "command": "/app/jre/bin/PortfolioPerformance",
+  "finish-args": [
+    "--socket=x11",
+    "--socket=wayland",
+    "--share=network",
+    "--filesystem=home",
+    "--device=dri",
+    "--env=PATH=/app/jre/bin:/usr/bin"
+  ],
+  "build-options": {
+    "append-path": "/usr/lib/sdk/openjdk11/bin"
+  },
+  "modules": [
+    {
+      "name": "openjdk",
+      "buildsystem": "simple",
+      "build-commands": [
+        "/usr/lib/sdk/openjdk11/install.sh"
+      ]
+    },
+
+    {
+      "name": "PortfolioPerformance",
+      "buildsystem": "simple",
+      "build-commands": [
+	"unzip plugins/name.abuchen.portfolio.ui_*.jar 'icons/*'",
+        "install -Dm644 icons/pp_16.png /app/share/icons/hicolor/16x16/apps/${FLATPAK_ID}.png",
+        "install -Dm644 icons/pp_48.png /app/share/icons/hicolor/48x48/apps/${FLATPAK_ID}.png",
+        "install -Dm644 icons/pp_256.png /app/share/icons/hicolor/256x256/apps/${FLATPAK_ID}.png",
+        "install -Dm644 icons/pp_512.png /app/share/icons/hicolor/512x512/apps/${FLATPAK_ID}.png",
+        "rm -rf icons",
+        "install -Dm644 info.portfolio_performance.PortfolioPerformance.appdata.xml /app/share/metainfo/info.portfolio_performance.PortfolioPerformance.appdata.xml",
+        "install -Dm 644 info.portfolio_performance.PortfolioPerformance.desktop /app/share/applications/info.portfolio_performance.PortfolioPerformance.desktop",
+        "mv * /app/jre/bin/"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/buchen/portfolio/releases/download/0.50.4/PortfolioPerformance-0.50.4-linux.gtk.x86_64.tar.gz",
+          "sha256": "64ec3767277af9f4ff676b6775156a03cb55be07aafd36dd11da0093dbd23349",
+          "x-checker-data": {
+	   "type": "json",
+           "url": "https://api.github.com/repos/buchen/portfolio/releases/latest",
+ 	   "version-query": ".tag_name",
+	   "url-query": ".assets[] | select(.name==\"PortfolioPerformance-\" + $version + \"-linux.gtk.x86_64.tar.gz\") | .browser_download_url"
+	  }
+        },
+        {
+          "type" : "file",
+          "path" : "info.portfolio_performance.PortfolioPerformance.desktop"
+        },
+        {
+          "type" : "file",
+          "path" : "info.portfolio_performance.PortfolioPerformance.appdata.xml"
```

Comment:
New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html

Reply:
I'm updating it. It might be worth updating the [App Requeirements](https://github.com/flathub/flathub/wiki/App-Requirements#appstream) as that is still refering to appstream.